### DISCUSSION TITLE
Uploader Component Parameters - Style (and others)

### DISCUSSION THREAD

**Discussion by lnarimatsu** [google/mesop#559] [Q&A] [answered]
“Could the Uploader Button be enhanced by adding a **style parameter**? This would allow us to customize its appearance. 

Additionally, could we consider incorporating all parameters that are currently available in the button component? This would provide greater flexibility and consistency across our components.

_Func Uploder_
![image](https://github.com/google/mesop/assets/11890782/68823a87-2557-4045-91f3-ed328695ad28)

_Func Button_
![image](https://github.com/google/mesop/assets/11890782/58e1e74b-efae-4714-b20e-6a2598464368)

**Comment by wwwillchen** [comment, marked as answer]
I filed an issue here: https://github.com/google/mesop/issues/568

In the short-term, you could create your own web component to do the uploader functionality and have full control over styling: https://google.github.io/mesop/web_components/
